openapi/platform/platform.yaml

```
@@ -5186,6 +5186,25 @@ components:
                         - businessLocations
               required:
                 - data
+            salesPerson:
+              type: object
+              properties:
```

We will want to mark the field as proposed until it is supported. Also we need a description that answers these questions:
What value(s) should be sent when creating an order, can it be updated, how is the value used in the platform, does it relate to commissions, what about SMB self created orders, can there be multiple salespeople?

```suggestion
              type: object
              description: |- 
                Status: `Proposed`
                
                The user who will be credited with the sale.....
              properties:
```

---

For this iteration I'm only making it a readonly field in this API. Will not be defined on orders created with the external api (could be changed in the future).

Wouldn't the existence of commissions be a partner specific detail?

---

I think that SSC has a revenue per salesperson report. You are right that we don't currently support tracking commissions in platform. 

Your description is good for the current slice. I would still like to know how the value currently gets populated in the platform. Partners that create orders by API will want to be able to set it in the next slice.